DRODLib/Character.cpp

```diff
@@ -3571,6 +3571,11 @@ void CCharacter::Process(
 			}
 			break;
 
+			case CCharacterCommand::CC_ResetOverrides: {
+				paramX = paramY = paramW = paramH = paramF = NO_OVERRIDE;
+			}
```

Comment:
I think we need
bProcessNextCommand = true;

Reply:
So we do. I've pushed a new commit to fix